cdap-ui/app/cdap/components/HttpExecutor/RequestHistoryTab/RequestRow/index.tsx

```diff
@@ -131,7 +131,11 @@ const RequestRowView: React.FC<IRequestRowProps> = ({ classes, request, onReques
 
   return (
     <div>
-      <div className={classes.requestRow} onClick={() => onRequestClick(request)}>
+      <div
+        data-cy={`request-row-${request.requestID.toLocaleString()}`}
```

Comment:
Do we need to have localized text in the `data-cy` field?

Reply:
Done